README.md

```diff
@@ -61,6 +61,51 @@ Implementation details
   * Optional runtime blinding which attempts to frustrate differential power analysis.
   * The precomputed tables add and eventually subtract points for which no known scalar (secret key) is known, preventing even an attacker with control over the secret key used to control the data internally.
 
+Obtaining and verifying
+-----------------------
+
+The git tag for each release (e.g. `v0.6.0`) is GPG-signed by one of the maintainers.
+For a fully verified build of this project, it is recommended to obtain this repository
+via git, obtain the GPG keys of the signing maintainer(s), and then verify the release
+tag's signature using git.
+
+This can be done with the following steps:
+
+1. Check the latest release on the [Releases
+   page](https://github.com/bitcoin-core/secp256k1/releases). Determine the signing GPG ID
+   by clicking the green icon next to the tag name. For example, in the case of v0.6.0,
+   this would be `4861DBF262123605`.
+    - Consult [SECURITY.md](./SECURITY.md) for a complete listing of maintainer keys,
```

Comment:
"per Github" assumes that the user has obtained the git repo from GitHub, which is likely but not the only possibility.